zaza/openstack/charm_tests/mysql/tests.py

```diff
@@ -71,6 +71,45 @@ def get_leaders_and_non_leaders(self):
                 self.non_leaders.append(unit)
         return self.leader, self.non_leaders
 
+    def get_cluster_status(self):
+        """Get cluster status.
+
+        Return cluster status dict from the cluster-status action or raise
+        assertion error.
+
+        :returns: Dictionary of cluster status
+        :rtype: dict
+        """
+        logging.info("Running cluster-status action")
+        action = zaza.model.run_action_on_leader(
+            self.application,
+            "cluster-status",
+            action_params={})
+        assert action.data.get("results") is not None, (
+            "Cluster status action failed: No results: {}"
+            .format(action.data))
+        assert action.data["results"].get("cluster-status") is not None, (
+            "Cluster status action failed: No cluster-status: {}"
+            .format(action.data))
+        return json.loads(action.data["results"]["cluster-status"])
+
+    def get_rw_primary_node(self):
```

Comment:
Where is it called?

Reply:
I was using this in an earlier version of the tests. It is currently uncalled, however, I think we will need it in the future. I can take it out, but I think it will be useful and I will wind up adding it again at a later point.